### PR TITLE
Meharshahani vi-team.md add

### PR DESCRIPTION
**Pull request by meharshahani**

### CODE REVIEW

**Comment by meharshahani** [comment]
@lmmrssa what's the change request for this?

**Comment by lmmrssa** [comment]
> @lmmrssa what's the change request for this?

You have changed navigation file

**Comment by meharshahani** [comment]
> Navigation file is not supposed to be changed

@lmmrssa how do i fix that?

**Comment by meharshahani** [comment]
I removed the change on the navigation page. I did not intend to do it! @lmmrssa

**Comment by lmmrssa** [comment]
On Hold till orientation

File: pages/vi/vi-team.md [hunk 1]
```diff
@@ -52,6 +52,7 @@
 |[sjkadali](profiles/sjkadali-edited.md)|2019-10-21|
 |[gyawaliamit7](profiles/gyawaliamit7.md)|2020-01-14|
 |[mattscodeans](profiles/mattscodeans.md)|2020-01-19|
+|[meharshahani](profiles/meharshahani.md)|2020-04-16|
```
Comment (lmmrssa) [inline pages/vi/vi-team.md:55]: this is looks good

**Pull request closed without merging**